### ISSUE TITLE
Stringtie: gffcompare

### ISSUE THREAD

**Issue by sschmeier** [MultiQC/MultiQC#828]
**Tool Details**
 - Name of tool: gffcompare
 - Tool homepage: [http://ccb.jhu.edu/software/stringtie/gff.shtml#gffcompare](http://ccb.jhu.edu/software/stringtie/gff.shtml#gffcompare)
 - Tool description: The tool can be used to compare, merge, annotate and estimate accuracy of one or more GFF files (the "query" files), when compared with a reference annotation (also provided as GFF/GTF).
 - Data suitable for MultiQC plot(s): Output file of statistics of coverage of the reference annotation in terms of precision and sensitivity but also number of calls. Could be barcharts, percentage and numbers

**Tool output:**
 - Log filename pattern: `*.stats`
 - File contents
    * [gff.stats.zip](https://github.com/ewels/MultiQC/files/2362973/gff.stats.zip)

**Comment by ewels** [comment]
NB: This is part of stringtie, so should be a stringtie module with submodule for gffcompare.

**Comment by sschmeier** [comment]
I agree. Would be great. The only useful info you get from stringtie is when run with -v (verbose)... many lines in the resulting stderr are not of interest. The last two lines hold value (see attached). 
[stringtie_verbose_tail.txt.zip](https://github.com/ewels/MultiQC/files/2363650/stringtie_verbose_tail.txt.zip)

**Comment by cgates** [comment]
Hi @sschmeier,
We would also be interested in a Stringtie module for MultiQC; wondered if there was progress on this?
cg

**Comment by ewels** [comment]
So the last two lines are this:
```
Total count of aligned fragments: 1.32538e+07
Fragment coverage length: 205.097
```

I guess the number of fragments is already easy to get from other tools, so it's the `Fragment coverage length` that's of interest? What does this actually mean?

I don't really see where we would get this information from:

> _Output file of statistics of coverage of the reference annotation in terms of precision and sensitivity but also number of calls. Could be barcharts, percentage and numbers_

**Comment by sschmeier** [comment]
The comments were made with regards to the gffcomapre output. For me the gffcompare output is more of interest: I assembled some tx and compare against a reference. See `gff.stats` file above.

**Comment by caspargross** [comment]
_gffcompare_ is now released as an independent tools with its own publication [PMC7222033](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7222033/) . I will start working on a module for this.

**Comment by caspargross** [comment]
Pull request:
https://github.com/ewels/MultiQC/pull/1253

**Comment by ewels** [comment]
Never closed this original issue sorry. GffCompare has its own module (#1253, v1.11) that parses these stats. Thanks again!
